ivy/functional/frontends/torch/utilities.py

```diff
@@ -0,0 +1,9 @@
+# global
+
+# local
+import ivy
+
+
+def result_type(*arg1):

```

Comment:
This function is a task in progress by another contributor # #2924 As you can see from this PR would you please take another function?